src/dotnet/Interop.cs

```diff
@@ -903,17 +904,17 @@ internal static void DisposeFullContext(ref FullContext FullContext)
             Byte[] ModificationDescriptorBytes)
         {
             fixed (Byte *S = SecurityDescriptorBytes)
-                fixed (Byte *M = ModificationDescriptorBytes)
-                {
-                    IntPtr SecurityDescriptor;
-                    Int32 Result = FspSetSecurityDescriptor(
-                        (IntPtr)S, SecurityInformation, (IntPtr)M, out SecurityDescriptor);
-                    if (0 > Result)
-                        return null;
-                    SecurityDescriptorBytes = MakeSecurityDescriptor(SecurityDescriptor);
-                    FspDeleteSecurityDescriptor(SecurityDescriptor, _FspSetSecurityDescriptorPtr);
-                    return SecurityDescriptorBytes;
-                }
+            fixed (Byte *M = ModificationDescriptorBytes)
+            {
+                IntPtr SecurityDescriptor;
+                Int32 Result = FspSetSecurityDescriptor(
+                    (IntPtr)S, SecurityInformation, (IntPtr)M, out SecurityDescriptor);
+                if (0 > Result)
+                    return null;
+                SecurityDescriptorBytes = MakeSecurityDescriptor(SecurityDescriptor);
+                FspDeleteSecurityDescriptor(SecurityDescriptor, _FspSetSecurityDescriptorPtr);
+                return SecurityDescriptorBytes;
+            }
         }
```

Comment:
Please fix tabs vs spaces here. WinFsp uses 4 space indendation and no tabs. (Or perhaps the file was erroneously using tabs before?)

Reply:
Quite sure i used spaces ... VisualStudio auto-format must have changed that... I'll correct that